### PR TITLE
Add standalone installation docs

### PR DESCRIPTION
**Pull request by pavithraes**
N/A

## Description

This pull request:

- Adds a new page with standalone installation docs (moved from quickstart section)
- Updates the introduction page with links to other docs pages providing a learning path

## Pull request checklist

- [x] Did you test this change locally?
- [x] Did you update the documentation (if required)?
- [ ] Did you add/update relevant tests for this change (if required)? -

### CODE REVIEW

File: docusaurus-docs/conda-store/how-tos/install-standalone.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 1
+description: How to install conda-store locally in standalone mode
+---
+
+# Standalone (local) installation
+
```
Comment (pavithraes) [inline docusaurus-docs/conda-store/how-tos/install-standalone.md:7]: Add a note that is an experimental feature

File: docusaurus-docs/conda-store/introduction.md [hunk 2]
```diff
@@ -9,46 +9,62 @@ description: Introduction to conda-store (core) documentation
...
+* [Descriptions for the artifacts created by conda-store][artifacts]
 
-<TabItem value="pip" label="pip" default>
+## Other installation options 💻
 
-```bash
-pip install conda-store conda-store-server
-```
-</TabItem>
+conda-store can also se installed using:
```
Comment (gabalafou) [inline docusaurus-docs/conda-store/introduction.md:30]:
```suggestion
conda-store can also be installed using:
```

**Review by gabalafou — COMMENTED** [review]
This looks great, just one question (see inline comments)

File: docusaurus-docs/conda-store/how-tos/install-standalone.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+   conda env create -f conda-store-server/environment-macos-dev.yaml
+   conda activate conda-store-server-dev
+   ```
+3. Install conda-store-server locally:
+   ```bach
+   pip install conda-store-server/.
+   ```
+</details>
+
+:::
```
[the remark this answers is not in the document]
Comment (pavithraes) [reply docusaurus-docs/conda-store/how-tos/install-standalone.md:41]: Good catch, it's not required anymore - will update. :)

Comment (pavithraes) [inline docusaurus-docs/conda-store/how-tos/install-standalone.md:41]:
```suggestion
:::note
The standalone mode is available in versions 2023.10.1 and later.
:::
```

File: docusaurus-docs/conda-store/how-tos/install-standalone.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 1
+description: How to install conda-store locally in standalone mode
+---
+
+# Standalone (local) installation
+
+The standalone mode is an **experimental feature**, and the quickest way to start using conda-store.
```
Comment (gabalafou) [inline docusaurus-docs/conda-store/how-tos/install-standalone.md:8]: Is it still considered an experimental feature? Or is this also old language that should go?

**Review by gabalafou — APPROVED** [review]
I'm going to approve this, but I've got one last question (inline).

**Pull request merged**